spec/support/queue_helpers.rb

```diff
@@ -32,14 +32,26 @@ def format_mtv_attorney_instructions(notes:, disposition:, hyperlinks: [])
   end
 
   def format_mtv_judge_instructions(notes:, disposition:, vacate_type: nil, hyperlink: nil)
-    parts = ["I am proceeding with a #{disposition_text[disposition.to_sym]}."]
-
-    parts += case disposition
-             when "granted", "partial"
-               ["This will be a #{vacate_types[vacate_type.to_sym]}", notes]
-             else
-               [notes, "\nHere is the hyperlink to the signed denial document", hyperlink]
-             end
+    parts = ["**Motion To Vacate:**  \n#{disposition_text[disposition]}\n"]
```

Comment:
```suggestion
    parts = ["**Motion To Vacate:**  \n#{disposition_text[disposition.to_sym].titlecase}\n"]
```

That should work to get the disposition into the correct format.